src/workflows/workflows.ts

```diff
@@ -303,4 +320,109 @@ export class Workflows {
       tradesApi: this.tradesApi,
     });
   }
+
+  /**
+   * IMX authorisation header functions
+   */
+  public async createProject(
+    ethSigner: EthSigner,
+    createProjectRequest: CreateProjectRequest,
+  ) {
+    const imxAuthHeaders = await generateIMXAuthorisationHeaders(ethSigner);
+
+    return this.projectsApi.createProject({
+      iMXSignature: imxAuthHeaders.signature,
+      iMXTimestamp: imxAuthHeaders.timestamp,
+      createProjectRequest,
+    });
+  }
+
+  public async getProject(ethSigner: EthSigner, id: string) {
+    const imxAuthHeaders = await generateIMXAuthorisationHeaders(ethSigner);
+
+    return this.projectsApi.getProject({
+      id,
+      iMXSignature: imxAuthHeaders.signature,
+      iMXTimestamp: imxAuthHeaders.timestamp,
+    });
+  }
+
+  public async getProjects(
+    ethSigner: EthSigner,
+    pageSize?: number,
+    cursor?: string,
+    orderBy?: string,
+    direction?: string,
+  ) {
+    const imxAuthHeaders = await generateIMXAuthorisationHeaders(ethSigner);
+
+    return this.projectsApi.getProjects({
+      iMXSignature: imxAuthHeaders.signature,
+      iMXTimestamp: imxAuthHeaders.timestamp,
+      pageSize,
+      cursor,
+      orderBy,
+      direction,
+    });
+  }
+
+  public async createCollection(
+    ethSigner: EthSigner,
```

Comment:
We seem to mix `ethSigner` with `signer`, let's stick with one. Maybe `ethSigner`?

Reply:
`ethSigner` makes more sense, it's more obvious what kind of signer is required.